sdk/provisioning/Azure.Provisioning/src/ModuleInfrastructure.cs

```diff
@@ -54,7 +52,7 @@ private void AddOutputsToModules()
 
         private Dictionary<Resource, List<Resource>> BuildResourceTree()
         {
-            var resources = _infrastructure.GetResources(true);
+            var resources = _infrastructure.GetResources(true).ToList();
```

Comment:
Why do we need ToList here?

Reply:
artifact from when I was iterating twice.